recipes/perf/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class Perf(ConanFile):
+    name = "perf"
+    description = "Linux profiling with performance counters"
+    topics = ("linux", "profiling")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://perf.wiki.kernel.org/index.php"
+    license = "GPL-2.0 WITH Linux-syscall-note"
+    settings = "os", "compiler", "build_type", "arch"
+
+    _source_subfolder = "source_subfolder"
+
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("perf is supported only on Linux")
+
+    def system_requirements(self):
+        installer = SystemPackageTool()
+        for package in [ "flex", "bison" ]:
```

Comment:
conan has a `flex` and `bison` package.
You can add that as a build requirement.